packages/orbit-components/src/utils/Slide/index.js

```diff
@@ -28,7 +28,7 @@ StyledSlide.defaultProps = {
 
 class Slide extends React.Component<Props, State> {
   state = {
-    maxHeight: 0,
+    maxHeight: typeof this.props.maxHeight !== "undefined" ? this.props.maxHeight : null,
```

Comment:
should it be null? or if false set to `0` maybe?

Reply:
Yep!